src/jmockit/assist/MockASTVisitor.java

```diff
@@ -74,22 +74,27 @@ public boolean visit(final MethodDeclaration node)
 
 		if ( mockedType != null )
 		{
-			boolean hasMockAnn = MockUtil.isMockMethod(meth);
-			boolean methodExists = findRealMethod(node, meth, mockedType) != null;
-
-			if (!hasMockAnn && methodExists )
-			{
-				addMarker(node.getName(), "Mocked method missing @Mock annotation", false);
-			}
+			boolean isClassInitMock = MockUtil.isClassInitializerMock(meth);
+
+			if (!isClassInitMock)
+			{		
+				boolean hasMockAnn = MockUtil.isMockMethod(meth);
+				boolean methodExists = findRealMethod(node, meth, mockedType) != null;
```

Comment:
Wouldn't it be better to do:
```java
methodExists = isClassInitMock  || findRealMethod(node, meth, mockedType) != null;
```
?
Then if the `@Mock` annotation is missing it will show a marker.